cli/flox/src/commands/mod.rs

```diff
@@ -350,6 +350,10 @@ impl FloxArgs {
             features: config.features.clone().unwrap_or_default(),
             verbosity: self.verbosity.to_i32(),
         };
+        debug!(
```

Comment:
We should wholesale switch over to `tracing` at this point, I think we just haven't fully made the switch yet so I have to resort to BARBARITIES like manual formatting of structured logging.

Reply:
We're using both `tracing::debug!` and `log::debug!` in a few places, which is messy but feels slightly better than deferring to a large migration, so we might as well do the right thing now:
```rust
        tracing::debug!(?flox.features, "feature flags");
```

Reply:
I would prefer we just make the cut over at some point: https://github.com/flox/flox/issues/2696